app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -1,13 +1,44 @@
 package com.example.myapplication;
 
+
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
 
-public class MainActivity extends AppCompatActivity {
+    private  Button nhan,dong;
```

Comment:
```suggestion
    private  Button nhan,dong;
```
sử dụng tiếng anh để đặt tnee cho các button
ví dụ Button buttonClose,